src/components/inputs/TextField.js

```diff
@@ -330,7 +326,7 @@ export default class TextField extends BaseInput {
     const {expandable, placeholder, placeholderTextColor, floatingPlaceholderColor, multiline} 
     = this.getThemeProps();
     const typography = this.getTypography();
-    const placeholderColor = this.getStateColor(placeholderTextColor || DEFAULT_COLOR_BY_STATE.default);
+    const placeholderColor = this.getStateColor(placeholderTextColor || Colors.grey30);
```

Comment:
Why to hard-code the color and not use the DEFAULT_COLOR_BY_STATE.default (that should be set to 'grey30' instead of 'grey10')?

Reply:
cause DEFAULT_COLOR_BY_STATE.default should be `grey10` and not `grey30`

Reply:
So why not hold a DEFAULT_PLACEHOLDER_COLOR_BY_STATE enum (like the DEFAULT_UNDERLINE_COLOR_BY_STATE) and get it from there instead of changing in 3 places?

Reply:
Cause technically the behavior is different. 
in line 329 I have 
`const placeholderColor = this.getStateColor(placeholderTextColor || Colors.grey30);`
in line 350 I have 
`outputRange: [placeholderColor, this.getStateColor(floatingPlaceholderColor || {focus: Colors.blue30, default: Colors.grey30})]`
which is different

and in line 500 I have this
`const placeholderColor = this.getStateColor(placeholderTextColor || Colors.grey30);`

The 2nd case is different than the other cause we want the color transition when the state changes from focused to not-focused. 
In the other cases (1+3), we don't want a different color for the different states. so I pass the `Colors.grey30`. If anything I can defined a single const for `DEFAULT_PLACEHOLDER_COLOR`

Reply:
I've defined `DEFAULT_PLACEHOLDER_COLOR` const